docs/experiments-msmarco-passage.md

```diff
@@ -230,7 +230,7 @@ We take the average of the scores across all queries (6980 in this case), and we
 You can find this run on the [MS MARCO Passage Ranking Leaderboard](https://microsoft.github.io/MSMARCO-Passage-Ranking-Submissions/leaderboard/) as the entry named "BM25 (Lucene8, tuned)", dated 2019/06/26.
 So you've just reproduced (part of) a leaderboard submission!
 
-We can also use the official TREC evaluation tool, `trec_eval`, to compute other metrics than MRR@10. 
+We can also use the official [TREC](https://en.wikipedia.org/wiki/Text_Retrieval_Conference) evaluation tool, `trec_eval`, to compute other metrics than MRR@10. 
```

Comment:
good call on the link, but let's link to official webpage? https://trec.nist.gov/

Reply:
Sure.